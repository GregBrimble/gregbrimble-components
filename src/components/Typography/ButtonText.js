import React from 'react';

import styled from 'styled-components';
import ThemeDefault from "../../themes/ThemeDefault";

const ButtonText = styled.span`
    color: ${(props) => props.theme.neutral['900']};
    font-family: 'Open Sans', sans-serif;
    font-size: 0.875rem;
    font-weight: 600;
    line-height: 2.25rem;
    text-transform: uppercase;
`;

ButtonText.defaultProps = {
    theme: ThemeDefault
};

export default ButtonText;
