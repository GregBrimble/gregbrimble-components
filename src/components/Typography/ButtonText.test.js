import React from "react"
import ButtonText from "./ButtonText"
import renderer from "react-test-renderer"

describe("ButtonText", () => {
  it("renders properly", () => {
    const tree = renderer.create(<ButtonText>Button Text</ButtonText>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
