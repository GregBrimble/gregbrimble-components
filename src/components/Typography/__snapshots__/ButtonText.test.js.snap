// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonText renders properly 1`] = `
<span
  className="sc-bdVaJa bNzTYa"
>
  Button Text
</span>
`;
